sale_order_customer_order_number/readme/DESCRIPTION.rst

```diff
@@ -0,0 +1,3 @@
+This module does the following:
+
+- Adds Customer Order Number to sale order line and propose the field to invoice line.
```

Comment:
```suggestion
- Adds Customer Order Number to sale order line and propose the field to invoice line.

This module can be used in case there are situations where a sales order contains multiple customer orders.
```